homeassistant/components/netatmo/__init__.py

```diff
@@ -93,21 +88,17 @@ async def async_setup(hass: HomeAssistant, config: ConfigType) -> bool:
     if DOMAIN not in config:
         return True
 
-    await async_import_client_credential(
+    config_flow.NetatmoFlowHandler.async_register_implementation(
         hass,
-        DOMAIN,
-        ClientCredential(
+        config_entry_oauth2_flow.LocalOAuth2Implementation(
+            hass,
+            DOMAIN,
             config[DOMAIN][CONF_CLIENT_ID],
             config[DOMAIN][CONF_CLIENT_SECRET],
+            OAUTH2_AUTHORIZE,
+            OAUTH2_TOKEN,
         ),
     )
-    _LOGGER.warning(
-        "Configuration of Netatmo integration in YAML is deprecated and "
-        "will be removed in a future release; Your existing configuration "
-        "(including OAuth Application Credentials) have been imported into "
-        "the UI automatically and can be safely removed from your "
-        "configuration.yaml file"
-    )
```

Comment:
Is this change on purpose? or does it come from a bad rebase?
It feels like this is reverting #71884

cc @allenporter

Reply:
That most certainly stems from a bad release. Good catch. I'll fix that.